### PR TITLE
Updated Channel App

### PR DESCRIPTION
**Pull request by theneims**

### CODE REVIEW

**Comment by ShadowRonin** [comment]
What are you trying to accomplish with this PR?

File: src/app/app.component.ts [hunk 1]
```diff
@@ -23,33 +29,37 @@ export class AppComponent implements OnInit, AfterViewChecked {
   ngOnInit() {
     initializeComplete();
     registerClickToDial(this.callSpecific);
-  }
-
-  ngAfterViewChecked() {
-    setAppHeight(document.body.clientHeight);
+    setAppHeight(200);
```
Comment (ShadowRonin) [inline src/app/app.component.ts]: Why was this changed? Is the height not being calculated correctly?

Comment (theneims) [reply src/app/app.component.ts]: It doesn't show up at all if you use the document.body.clientHeight. I tried with it and with the 200 and it only showed up with the 200.
